core/src/modules/configLoaders/destination.ts

```diff
@@ -77,6 +84,10 @@ export async function loadDestination(
     await destination.trackGroup(group);
   }
 
+  if (destination.state === "deleted") {
+    await destination.exportGroupMembers(true);
+  }
```

Comment:
Were we not doing a final export before?  If not... this should be added in a test.